src/KafkaFlow.Serializer.JsonCore/KafkaFlow.Serializer.JsonCore.csproj

```diff
@@ -13,5 +13,6 @@
 
     <ItemGroup>
         <ProjectReference Include="..\KafkaFlow.Abstractions\KafkaFlow.Abstractions.csproj" />
+        <ProjectReference Include="..\KafkaFlow\KafkaFlow.csproj" />
```

Comment:
I think this dependency is no more needed

Reply:
Yes, same for Newtonsoft and ProtobufNet projects 👍

Reply:
Done